test/ComponentRouter-test.js

```diff
@@ -1,23 +1,160 @@
+import React from 'react/addons';
+import createComponent from './utils/createComponent';
+import FooBar from '../src/example/FooBar/FooBar';
```

Comment:
Just create an empty component here in place to avoid foobar be added as untested dep. Like you have done with Chart